src/report-filter-controls.tsx

```diff
@@ -81,18 +81,28 @@ function ReportFilterControls({
   }
 
   return (
-    <div>
+    <div className="usa-form">
       <form ref={formRef} onChange={update}>
         <div>
           <label>
             Start
-            <input type="date" name="start" value={yearMonthDayFormat(start)} />
+            <input
+              type="date"
+              name="start"
+              value={yearMonthDayFormat(start)}
+              className="usa-input"
```

Comment:
I don't know if we'd want to use it, especially since a lot of the JavaScript-heavy design system components don't work so well in React applications, but there is a design system date picker we could use:

https://designsystem.digital.gov/components/date-picker/

Reply:
The native one in Chrome seems to be OK for now?
<img width="358" alt="Screen Shot 2021-09-13 at 2 21 16 PM" src="https://user-images.githubusercontent.com/458784/133158043-3862077b-7bf2-4e29-bb84-4bbc8f70757c.png">